simphony_ui/couple_CFDDEM.py

```diff
@@ -85,7 +86,7 @@
 # number of bins for velo profile generation
 Nbins_lmp = Nbins_OF
 
-restart_file = "DEM_input.dat"
+restart_file = osp.abspath(osp.join("simphony_ui", "DEM_input.dat"))
```

Comment:
will not work. abspath absolutizes something relative to the current path anyway.
```
>>> os.path.abspath("foo")
'/home/stefano/simphony-remote/remoteappmanager/jupyterhub/foo'
>>> os.chdir("/tmp")
>>> os.path.abspath("foo")
'/tmp/foo'
>>> 
```

you need __file__ to find the file relative to the current py file.